docs/getting-started/linux-quickstart.md

```
@@ -0,0 +1,118 @@
+sudo apt update
+sudo apt install shed-server
+```
+
+This installs `shed` + `shed-server`, generates a default Firecracker config at
+`/etc/shed/server.yaml`, and registers a systemd service.
+
+??? note "Alternative: install the `.deb` directly"
+    Without the apt repo, grab the `.deb` from the
+    [releases page](https://github.com/charliek/shed/releases):
+
+    ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)